packages/remediations/src/utils.js

```diff
@@ -14,3 +14,5 @@ export function getGroup () {
 export function remediationUrl (id) {
     return urijs(document.baseURI).segment(getGroup()).segment('remediations').segment(id).toString();
 }
+
+export const pluralazie = (text = '', count = 0) => `${text}${count !== 1 ? 's' : ''}`;
```

Comment:
Maybe we should start using react-intl for plurals? It is in this package anyway.